documentation/decisions/003_component_api_style.md

```diff
@@ -0,0 +1,44 @@
+---
+status: Accepted
+date: 2023-02-03
+deciders: stephenjwatkins, OskiTheCoder, kevinalexliu
+---
+
+# Design component APIs based on common patterns that exist across popular libraries.
```

Comment:
curious if you think we can make this heading a little more succinct and without it being in sentence case for consistency with other ADRs?

```suggestion
# Use standard conventions for component APIs
```

Reply:
agree